writings/2022-08-23-flashbots-protect-merge-beyond.mdx

```diff
@@ -0,0 +1,51 @@
+---
+slug: flashbots-protect-merge-and-beyond
+title: Flashbots Protect -- The Merge & Beyond
+authors: [angela]
+tags: [protect]
+# image: img/og_forest.jpg
+hide_table_of_contents: false
+---
+
+# Flashbots Protect -- The Merge & Beyond
+
+TL;DR: It has been a great year for Flashbots Protect! Post-merge, all transactions sent to Protect will have frontrunning and revert protection. But due to how mev-boost will be launched at the merge, Flashbots Protect will be down for at least 2 hours after the merge. If you rely on Flashbots Protect as a user or application developer then please plan accordingly.
```

Comment:
"But due to how mev-boost will be launched at the merge, Flashbots Protect will be down for at least 2 hours after the merge" -> 2 hours? says 1 hour later